linera-service/src/linera.rs

```diff
@@ -1023,18 +1017,12 @@ where
                 context.save_wallet();
             }
 
-            Watch { chain_ids, raw } => {
-                debug!("Watching for notifications for chains: {:?}", &chain_ids);
-                let chain_id = match chain_ids.get(0) {
-                    None => {
-                        warn!("No chains specified, exiting...");
-                        return Ok(());
-                    }
-                    Some(chain_id) => chain_id,
-                };
-                let mut chain_client = context.make_chain_client(storage, *chain_id);
+            Watch { chain_id, raw } => {
+                let mut chain_client = context.make_chain_client(storage, chain_id);
+                let chain_id = chain_client.chain_id();
+                debug!("Watching for notifications for chain {:?}", chain_id);
                 let mut tracker = NotificationTracker::default();
-                let mut notification_stream = chain_client.subscribe_all(chain_ids).await?;
+                let mut notification_stream = chain_client.connect().await?;
                 while let Some(notification) = notification_stream.next().await {
                     if raw {
                         info!("{:?}", notification);
```

Comment:
Should this be `println!`? Otherwise it'll show nothing if log level is e.g. WARN.

Reply:
Makes sense